src/components/Icons/FeetIcon.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+import React from "react";
+import Icon from "@/components/Icons/Icon";
+import { faShoePrints } from "@fortawesome/free-solid-svg-icons";
+import styled from "styled-components";
+
+interface Props {
+    collectionPoint: string;
+}
+
+const StyledIcon = styled(Icon)`
+    background-color: red;
```

Comment:
Can you have default colours for each of the icons as per the Jira ticket? eg Congestion Charge is red, Speech Bubble is blue, ...
(we can [and likely will] change the exact shades later, but I'd like to have _something_ in from the start!)